queue/src/fiskaltrust.Middleware.Storage.AzureTableStorage/Repositories/AzureTableStorageQueueItemRepository.cs

```diff
@@ -97,15 +97,30 @@ public override async Task InsertAsync(ftQueueItem storageEntity)
             await _receiptReferenceIndexRepository.InsertAsync(new ReceiptReferenceIndex { cbReceiptReference = storageEntity.cbReceiptReference, ftQueueItemId = storageEntity.ftQueueItemId });
         }
 
-        public override async Task InsertOrUpdateAsync(ftQueueItem storageEntity)
+
+        public async Task InsertOrUpdateAsync(ftQueueItem storageEntity)
         {
-            await base.InsertOrUpdateAsync(storageEntity);
+            EntityUpdated(storageEntity);
+            var entity = MapToAzureEntity(storageEntity);
```

Comment:
IMO we should add a comment or a test somewhere to make sure that we are keeping ftQueueItem with partitionkey of queuerow ( different to al others).